docx/oxml/table.py

```diff
@@ -759,6 +759,7 @@ class CT_TcPr(BaseOxmlElement):
         'w:cellDel', 'w:cellMerge', 'w:tcPrChange'
     )
     tcW = ZeroOrOne('w:tcW', successors=_tag_seq[2:])
+    tcBorders = ZeroOrOne('w:tcBorders', successors=_tag_seq[6:])
```

Comment:
Why do we need this?

Reply:
we don't it was probably a leftover from different branch, thanks!